platforms/macos/src/subclass.rs

```diff
@@ -171,6 +184,25 @@ impl SubclassingAdapter {
         }
     }
 
+    /// Create an adapter that dynamically subclasses the content view
+    /// of the specified window. Panics if the specified window doesn't
```

Comment:
I'd put the note about panics in a dedicated `# Panics` section, like the stdlib does.